sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -610,7 +610,8 @@ internal static LinkedEntityMatch LinkedEntityMatch(double confidenceScore = def
                 recognizeEntitiesActionResults.ToList(),
                 recognizePiiEntitiesActionResults.ToList(),
                 recognizeLinkedEntitiesActionsResults.ToList(),
-                analyzeSentimentActionsResults.ToList());
+                analyzeSentimentActionsResults.ToList(),
+                null); // TODO: include extractive summarization API (https://github.com/Azure/azure-sdk-for-net/issues/22836).
```

Comment:
I understand you want to do it in another PR but I am not sure why not doing it here?

Reply:
Now that you mention it, all changes would happen in the `TextAnalyticsModelFactory` class, so it probably wouldn't be a big deal. Just wanted to avoid adding extra changes to this PR (it's already larger than I was expecting). Also, the factory is a .NET-specific type as far as I know. I can update it in a separate PR and tag only Mariana so folks from other languages don't need to worry about it.

Reply:
Sounds good.